packages/shared/src/components/MainLayout.tsx

```diff
@@ -46,31 +43,22 @@ export default function MainLayout({
   children,
   showOnlyLogo,
   responsive = true,
-  showRank,
   greeting,
-  mainPage,
-  additionalButtons,
   onLogoClick,
   onShowDndClick,
 }: MainLayoutProps): ReactElement {
   const { windowLoaded } = useContext(ProgressiveEnhancementContext);
   const { user, showLogin, loadingUser } = useContext(AuthContext);
+  const { openSidebar, toggleOpenSidebar } = useContext(SettingsContext);
   const [showGreeting, setShowGreeting] = useState(false);
 
-  const afterBookmarkButtons = (
-    <>
-      {additionalButtons}
-      {mainPage && <FeedSettingsButton />}
-    </>
-  );
-
   return (
     <>
       {!responsive && <PromotionalBanner />}
       <header
         className={`${
           styles.header
-        } relative flex items-center px-4 border-b border-theme-divider-tertiary tablet:px-8 laptop:px-4 ${
+        } relative flex items-center px-4 border-b border-theme-divider-tertiary tablet:px-8 laptop:px-4 py-2 ${
```

Comment:
This is just an improvement in the existing code, I think we can turn this to using `classnames` instead of template strings.